src/plugins/data_source_management/public/components/edit_data_source/components/edit_form/edit_data_source_form.tsx

```diff
@@ -100,11 +101,7 @@ export class EditDataSourceForm extends React.Component<EditDataSourceProps, Edi
       auth: {
         type: initialSelectedAuthMethod?.name,
         credentials: {
-          username: '',
-          password: '',
-          region: '',
-          accessKey: '',
-          secretKey: '',
+          ...initialSelectedAuthMethod?.crendentialFormField,
```

Comment:
typo: `crendential`

Reply:
Thanks for the comment and good catch

I saw this name is wildly used :(  
Since we need to deliver this change in a time manner to catch the release, so let's keep it for now.
But we will surly add it into the backlog and fix it!

Appreciate again~

Reply:
Typo updated. Thanks